portfolio/src/main/webapp/script.js

```diff
@@ -114,39 +150,107 @@ function initMap() {
       position: new google.maps.LatLng(favoriteSpots[i][1], favoriteSpots[i][2]),
       map: map
     });
+
     marker.addListener('click', function() {
       infowindow.open(map, marker);
     });
   }
+
+  map.addListener('click', (event) => {
+    createMarkerForEdit(event.latLng.lat(), event.latLng.lng());
+  });
+
+  // Add user-entered markers
+  fetchMarkers();
+
+  let psaMap = new google.maps.Map(document.getElementById('psa-map'), mapOptions);
 }
 
-/**
- * Fetches text from the server ArrayList and adds them to the DOM.
+/*
+ * Fetch user markers from datastore and add them to map
```

Comment:
Comments typically have an implied "This method/function" or "This class" in front. Consider updating this to "Fetches user markers from datastore and adds them to map."